src/bot.ts

```diff
@@ -141,9 +144,16 @@ export function createBot(config: BotConfig) {
       return path.resolve(config.dataDir, `${content}.mp3`);
     }
     if (config.myInstantsEnabled && content.startsWith("instant")) {
-      const sound = encodeURIComponent(content.substr("instant".length).trim().substr(0, 256))
+      const sound = encodeURIComponent(content.slice("instant".length).trim().slice(0, 256))
       return `https://www.myinstants.com/media/sounds/${sound}.mp3`;
     }
+    if (config.mediawikiCDNEnabled && content.startsWith("fandom")) {
+      const [domain, filename] = content.slice("fandom".length).trim().split(/\s+/);
```

Comment:
Are the any `filename`s containing spaces?

Reply:
FWIK mediawiki converts whitespaces of all uploaded files to underscores.
[Mentioned here in a discussion in support desk.](https://www.mediawiki.org/wiki/Topic:Vu1get4npnuuatlp)

Reply:
sounds good